README.md

```diff
@@ -9,7 +9,7 @@ As a fully decentralized dataspace is hard to imagine, the MVD also serves the p
 
 ## Documentation
 
-Developer documentation can be found under [docs/developer](docs/developer), where the main concepts and decisions are captured as [decision records](docs/developer/decision-records/).
+Developer documentation can be found under [docs/developer](docs/developer), where the main concepts and decisions are captured as [decision records](docs/developer/decision-records).
```

Comment:
We need the `/` or a `/README.md` because this file is rendered with GitHub pages and will be broken when linking to a folder. Talking of... `docs/developer` should be changed accordingly, as it currently leads to a 404, see https://eclipse-edc.github.io/docs/#/submodule/MinimumViableDataspace/: https://eclipse-edc.github.io/docs/#/submodule/MinimumViableDataspace/docs/developer (currently) vs. https://eclipse-edc.github.io/docs/#/submodule/MinimumViableDataspace/docs/developer/ or https://eclipse-edc.github.io/docs/#/submodule/MinimumViableDataspace/docs/developer/README.md

Reply:
Please change to 
```
Developer documentation can be found under [docs/developer](docs/developer/), where the main concepts and decisions are captured as [decision records](docs/developer/decision-records/).
```
as requested.